src/core/io/src/4C_io_input_parameter_container.templates.hpp

```diff
@@ -0,0 +1,46 @@
+// This file is part of 4C multiphysics licensed under the
+// GNU Lesser General Public License v3.0 or later.
+//
+// See the LICENSE.md file in the top-level for license information.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#ifndef FOUR_C_IO_INPUT_PARAMETER_CONTAINER_TEMPLATES_HPP
+#define FOUR_C_IO_INPUT_PARAMETER_CONTAINER_TEMPLATES_HPP
+
+#include "4C_config.hpp"
+
+#include "4C_io_input_parameter_container.hpp"
+
+#include <Teuchos_ParameterList.hpp>
+
+FOUR_C_NAMESPACE_OPEN
+
+// The add() function requires expensive to include Teuchos headers, but it is rarely needed, since
```

Comment:
This comment is somewhat unclear 😅

```suggestion
// The add() function requires expensive to include Teuchos headers, but it is rarely needed, since
```

Reply:
Is it clearer now?

Reply:
Oh thats meant - sure! Thanks:)